cmd/src/gateway_benchmark.go

```diff
@@ -0,0 +1,258 @@
+package main
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+const (
+	defaultRequestCount = 100
+	blue                = "\033[34m"
+	green       = "\033[32m"
+	yellow      = "\033[33m"
+	red         = "\033[31m"
+	reset       = "\033[0m"
+)
+
+func init() {
+	usage := `
+'src gateway benchmark' runs performance benchmarks against Cody Gateway endpoints.
+
+Usage:
+
+    src gateway benchmark [flags]
+
+Examples:
+
+    $ src gateway benchmark
+    $ src gateway benchmark --requests 50
+`
+
+	flagSet := flag.NewFlagSet("benchmark", flag.ExitOnError)
+
+	var (
+		requestCount = flagSet.Int("requests", defaultRequestCount, "Number of requests to make per endpoint")
+	)
+
+	handler := func(args []string) error {
+		if err := flagSet.Parse(args); err != nil {
+			return err
+		}
+
+		if len(flagSet.Args()) != 0 {
+			return cmderrors.Usage("additional arguments not allowed")
+		}
+
+		// Create HTTP client with TLS skip verify
+		client := &http.Client{
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			},
+		}
```

Comment:
I think it's fine if we have a CLI option for `--insecure` to disable TLS for testing against local dev, but this wouldn't match a real prod environment so let's make sure we have TLS verification enabled by default here

Reply:
I think we don't need it anyway, the script works without it locally. Removed.